docs/db/refactor_platform_rs.dbml

```diff
@@ -0,0 +1,130 @@
+  logo varchar [note: 'A URI pointing to the organization\'s logo icon file']
+  created_at timestamp 
+}
+
+// Coaching relationship type belonging to the refactor_platform schema
+// from the perspective of the coach
+enum refactor_platform.coaching_relationship {
+  client
+  direct_report
+}
+
+// Declares a coach <--> coachee coaching relationship
+// E.g. Manager <--> Software Engineer direct report
+// E.g. Independent coach <--> Software Engineer client
+Table refactor_platform.coaches_coachees {
+  id integer [primary key, unique, not null, increment]
+  organization_id integer [note: 'The organization associated with this coaching relationship']
+  coach_id integer [note: 'The coach associated with this coaching relationship']
+  coachee_id integer [note: 'The coachee associated with this coaching relationship']
+  relationship coaching_relationship 
+}
+
+Table refactor_platform.users {
+  id integer [primary key, unique, not null, increment]
+  email varchar
+  first_name varchar
+  last_name varchar
+  display_name varchar [note: 'If a user wants to go by something other than first & last names']
+  password varchar
+  timezone timezone
+  github_username varchar // Specifically GH for now, can generalize later
+  github_profile_url varchar
+  created_at timestamp
+}
+
+Table refactor_platform.coachees {
+  id integer [primary key, unique, not null, increment]
+  user_id integer
+  organization_id integer
+}
+
+Table refactor_platform.coaches {
+  id integer [primary key, unique, not null, increment]
+  user_id integer
+  organization_id integer
+}
+
+Table refactor_platform.coaching_sessions {
+  id integer [primary key, unique, not null, increment]
+  coaches_coachees_id integer [note: 'The coaching relationship (i.e. what coach & coachee under what organization) associated with this coaching session']
+  date timestamp [note: 'The date and time of a session']
+  timezone timezone [note: 'The baseline timezone used for the `date` field']
+  created_at timestamp
+}
+
+Table refactor_platform.overarching_goals {
+  id integer [primary key, unique, not null, increment]
+  coaching_session_id integer [note: 'The coaching session that an overarching goal is associated with']
+  title varchar [note: 'A short description of an overarching goal']
+  details varchar [note: 'A long description of an overarching goal']
+  completed boolean // May be unnecessary if there's a valid completed_at timestamp
+  completed_at timestamp [note: 'The date and time an overarching goal was completed']
+  completed_by_id integer [note: 'The user that successfully completed an overarching goal']
+  created_at timestamp
+  updated_at timestamp [note: 'The last date and time an overarching goal\'s fields were changed']
+}
+
+Table refactor_platform.notes {
+  id integer [primary key, unique, not null, increment]
+  coaching_session_id integer
+  body varchar [note: 'Main text of the note supporting Markdown']
```

Comment:
This and some of the other larger text fields (Ex `agreements.details` below) might be good candidates for the `text` type.

Reply:
@calebbourg great suggestion, I agree.